Docs/developer_guide/extensions.md

```diff
@@ -16,10 +16,9 @@ If you have developed a script or module that you would like to share with other
 ## Build an extension
 
 :::{note}
+To compile extensions containing C++-implemented modules, it's essential to [build Slicer from source](build_instructions/index.md) on your local machine. These extensions cannot be compiled against a binary download. However, if your modules are solely developed in Python, there's no need to build the extension.
 
-To build extensions that contain modules implemented in C++, you need to [build Slicer from source](build_instructions/index.md) on your machine; they cannot be built against a binary download.
-If developing modules in Python only, then it is not necessary to build the extension.
-
+Once your work is ready for sharing, refer to [](#extension-distribute) for distribution guidelines.
```

Comment:
```suggestion
Once your work is ready for sharing, refer to [](#extension-distribute) for guidelines.
```

Since the section is related to distribution, there is no need to repeat the word "distribution":

<img width=75% src="https://github.com/Slicer/Slicer/assets/219043/2b031e9a-3777-44b2-ab7e-24d32308cc1f"/>